public/css/main.stylesheet.css

```diff
@@ -682,6 +702,9 @@ div.icon img {
 
 .moduleProperty input {
   color: #fff;
+  margin-top: 7px !important;
+  border: 1px solid white !important;
+  padding: 0 5px !important;
```

Comment:
Do we really need to use `!important` for all these? 
Also we are applying these in `input`.
can cause lot of issues when we need to override properties in future.

Reply:
removed !important, there were some overriding styles, now fixed

Reply:
@abstrekt You forgot to push the changes? I don't see any new changes.

Reply:
pushed the changes